src/decorators/relationships.js

```diff
@@ -132,10 +136,9 @@ export function getRelatedRecords (record, property, modelType = null) {
       }
     }
 
-    record.relationships = {
-      ...relationships,
+    record.cachedRelationships = {
+      ...cachedRelationships,
```

Comment:
where does the initial value of `cachedRelationships` come from?

Reply:
> where does the initial value of `cachedRelationships` come from?

`x = {...x}` works fine when x is undefined or null